cmd/deploy.go

```diff
@@ -150,36 +139,15 @@ func deployFn(_ *cobra.Command, _ []string) error {
 		SetSkipPostDeploy(skipPostDeploy).
 		SetSkipLabDirFileACLs(skipLabDirFileACLs)
 
-	containers, err := c.Deploy(ctx, deploymentOptions)
+	containers, err := c.Deploy(cobraCmd.Context(), deploymentOptions)
 	if err != nil {
 		return err
 	}
 
+	// TODO
 	// log new version availability info if ready
-	version.NewVerNotification(vCh)
+	// version.NewVerNotification(vCh)
 
 	// print table summary
 	return inspect.PrintContainerInspect(containers, deployFormat)
 }
-
-// setupCTRLCHandler sets-up the handler for CTRL-C
```

Comment:
replaced by cmd/context.go basically same thing, just copypastad from c9s for the new one